### PR TITLE
development <- main sync request

### PR DESCRIPTION
**Pull request by brandonv98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,180 @@
...
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+<i>note: this is a template and needs to be updated.</i>
+
```
Comment (gabesmith08) [inline README.md:19]: keep line spaces consistent, seems to be 3 lines of space everywhere else

Comment (stamatim) [reply README.md:19]: For which part do you mean?

File: package.json [hunk 2]
```diff
@@ -23,7 +23,10 @@
     "express": "^4.16.4",
     "nodemon": "^1.18.9"
   },
-  "author": "Brandon VanCamp",
+  "authors": [
+    "Brandon VanCamp",
+    "Stamati Morellas"
+  ],
```
Comment (stamatim) [inline package.json:29]: People should add their names here as needed

**Review by stamatim — COMMENTED** [review]
Looks good. Haven't tested all of the endpoints yet so I am going to hold off on approving for now. Can find sometime later to test perhaps.

File: README.md (hunk 1 above)
Comment (brandonv98) [reply README.md:19]: I'm not following either @gabesmith08

Comment (gabesmith08) [reply README.md:19]: idk why it highlighted like that hahahah

File: README.md [hunk 3]
```diff
@@ -1,4 +1,180 @@
...
+
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
```
Comment (gabesmith08) [inline README.md:11]: this part is only one space

**Pull request closed without merging**